app/src/test/java/org/simple/clinic/enterotp/BruteForceOtpEntryProtectionTest.kt

```diff
@@ -64,7 +64,7 @@ class BruteForceOtpEntryProtectionTest {
     bruteForceOtpEntryProtection.incrementFailedOtpAttempt()
 
     verify(preferenceState).set(BruteForceOtpProtectionState(
-        failedLoginOtpAttempt = config.limitOfFailedAttempts + 1,
+        failedLoginOtpAttempt = config.limitOfFailedAttempts,
```

Comment:
Hmm, how is reducing the number of limits reached sufficient to test the new behaviour? We added a new branch, but no new tests have been added, so I assume we are missing at least one more test to verify the new branch, right?

Reply:
So +1 was the problem. After the user has reached the max attempts, the blocked at time wasn't getting updated but the number of attempts was getting updated. I think the only thing we need to update in this test is the test name.
  fun `when otp limit was already reached then the blocked-at time and the number of attempts should not be updated`

Reply:
If you look at `BruteForceOtpEntryProtectionTest`, all three scenarios are covered.